app/sequencescape/sequencescape/api/v2/tube.rb

```diff
@@ -51,6 +51,11 @@ def self.find_all(params)
 
   delegate :requests_as_source, to: :receptacle
 
+  # Mocked out for the time being
+  def in_progress_submission_uuids(*)
```

Comment:
If the method for `v1` previously didn’t accept any params, would the `(*)` change the definition of the method? (just incase it is being used somewhere)